lib/src/react_client/synthetic_event_wrappers.dart

```diff
@@ -59,15 +59,6 @@ class SyntheticFocusEvent extends SyntheticEvent {
 @anonymous
 class SyntheticFormEvent extends SyntheticEvent {}
 
-@JS()
-@anonymous
-class SyntheticDataTransfer {
```

Comment:
Should this be deprecated instead of removed? (with a comment about it not existing on the JS layer)

Reply:
No, this class (and the rest of this file) was not previously exported

Reply:
Sweet.